### ISSUE TITLE
Loss and val_loss improvement

### ISSUE THREAD

**Issue by dikshaaswal** [david8862/keras-YOLOv3-model-set#104]
Thanks for the wonderful code.
I am trying to use diou loss, but after ep010-loss3.02-val-loss3.123, there is no improvement. How to fix this.

**Comment by david8862** [comment]
@dikshaaswal  not sure about your training detail. Have you ever tried to reduce the learning rate?

**Comment by dikshaaswal** [comment]
The training details are same as your's given in the code except using DIoU loss.

**Comment by rootadminWalker** [comment]
@dikshaaswal Have you found the solution for loss improve? cuz I'm experiencing the same problem right now
